letsencrypt/data/run.sh

```diff
@@ -73,6 +76,19 @@ elif bashio::config.exists 'dns.google_creds'; then
       bashio::log.info "Google Credentials File doesnt exists in folder share."
     fi
     PROVIDER_ARGUMENTS+=("--${DNS_PROVIDER}" "--${DNS_PROVIDER}-credentials" "/data/${GOOGLE_CREDS}")
+#Netcup
+elif bashio::config.exists 'dns.netcup_customer_id' && bashio::config.exists 'dns.netcup_api_key' && bashio::config.exists 'dns.netcup_api_password'; then
+    if bashio::config.exists 'dns.netcup_propagation_seconds'; then
+      bashio::log.info "propagation time found for netcup"
+      NETCUP_DNS_PROPAGATION_SECONDS="$(bashio::config 'dns.netcup_propagation_seconds')"
+    else
+      NETCUP_DNS_PROPAGATION_SECONDS=30
+      bashio::log.info "no propagation time found for netcup, using default value"
+    fi
+
+    export NETCUP_DNS_PROPAGATION_SECONDS
```

Comment:
is not needed. Did we set this as a parameter, so why should be also inside env?